client/src/app/shared/models/motions/motion.ts

```diff
@@ -100,31 +98,43 @@ export class Motion extends BaseModel {
     }
 
     /**
-     * sets the workflow_id and the workflow
+     * delegate the isStateContainId-function to a workflow
+     * @param workflow
      */
-    initDataStoreValues() {
-        const motionsWorkflowConfig = this.DS.filter(Config, config => config.key === 'motions_workflow')[0] as Config;
-        if (motionsWorkflowConfig) {
-            this.workflow_id = +motionsWorkflowConfig.value;
+    isStateIdMatchWorkflow(workflow: Workflow): boolean {
```

Comment:
This function is not necessary. You can just use `workflow.isStateContainId(this.state_id)` at the required places.